### ISSUE TITLE
Arrays inside arrays

### ISSUE THREAD

**Issue by Rochet2** [fotinakis/swagger-blocks#75]
Hello.

I was trying to document my ruby rails webserver that returns following JSON

``` json
{
  "lists": [
    [
      {
        "name": "test",
        "urls": [ ]
      }
    ]
  ]
}
```

I am using swagger-blocks and I could not create such a structure that has two or more arrays inside each other. 
I tried several ways. Here is something I have now, but obviously it did not work.

``` ruby
class MyClass < ActiveRecord::Base
  include Swagger::Blocks

  swagger_schema :MyOutput do
    property :lists, type: :array do
      items type: :array do
        items type: :object do
          property :name, type: :string, example: "test"
          property :urls, type: :array do
            items type: :string
          end
        end
      end
    end
  end

  # ...
end
```

Am I doing something wrong or is this not possible?

**Comment by lorainekv** [comment]
Swagger 2.0 [appears to support](https://github.com/OAI/OpenAPI-Specification/issues/273#issuecomment-74270651) nesting items within items for 2D arrays.  Plans to match this functionality?

**Comment by fotinakis** [comment]
Hey @lorainekv, would you be interested in taking on the implementation of this and sending a PR? It should be relatively straightforward, I think all one needs to do is: add an `items` method to the ItemsNode [here](https://github.com/fotinakis/swagger-blocks/blob/4a5d33939e49f4417f1fc52896a3c30b69a5b27c/lib/swagger/blocks/nodes/items_node.rb#L11) (the method will be almost exactly like [this](https://github.com/fotinakis/swagger-blocks/blob/4a5d33939e49f4417f1fc52896a3c30b69a5b27c/lib/swagger/blocks/nodes/property_node.rb#L5)), as well as adding a test for it.

**Comment by lorainekv** [comment]
Sure thing, thanks for the helpful response! [Here's a PR](https://github.com/fotinakis/swagger-blocks/pull/99)

PS thank you for this gem :)

**Comment by fotinakis** [comment]
Fixed by #99 and released in v2.0.1 of the gem. Thanks!
